### PR TITLE
Use npm as publishing registry.

### LINKED ISSUE #20: Think about publishing packages to NPM registry instead of GitHub packages
As I found out today it is not possible at the moment to read npm packages from the GitHub Packages registry without a personal access token. Even for public repositories/packages...

This is very unfortunate...

By digging deeper I found that this will probably not be solved in the near future: https://github.community/t/download-from-github-package-registry-without-authentication/14407

So maybe we should start publishing to the real NPM registry ?

This would also resolve #17 

cc @quarkus @lukas-schardt @KinG-DE

### PR DESCRIPTION
**Pull request by lukas-schardt**
Closes #20 .

Removed the references to the github package registry.

~~Still TODO: We need to save the NPM publishing token inside the NPM_TOKEN secret. I don't seem to have enough rights.~~

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 node_modules/
 test/coverage
+.idea
+*.iml
```
Comment (eddyloewen) [inline .gitignore:4]: @lukas-schardt I'm not a big fan of these editor specific files in here. I would encourage everyone to put these in a global .gitignore on your computer.

File: README.md [hunk 2]
```diff
@@ -10,11 +10,6 @@ Simple and lightweight base classes for web components with a beautiful API
 
 #### Installation
```
Comment (eddyloewen) [inline README.md:11]: @quarkus How should we notify people that are using element-js already? A note in the changelog/release notes should be fine?

**Comment by eddyloewen** [comment]
What about the other packages? Will we have to move them all over before we can start installing element-js?
It will probably not be possible to install packages with the same namespace from two different registries - right?

**Comment by lukas-schardt** [comment]
@eddyloewen I don't think so, only if we are updating the element-js version on the other package. The currently existing GitHub Package should still be usable by the other packages.

**Comment by eddyloewen** [comment]
> @eddyloewen I don't think so, only if we are updating the element-js version on the other package. The currently existing GitHub Package should still be usable by the other packages.

Yes that is true! But if we would update element-js further and I wanted to use a newer version it will only be available on the NPM registry. But thats ok! Eventually we will have to move all the other packages to NPM as well. Rather sooner than later!

File: README.md (hunk 2 above)
Comment (quarkus) [reply README.md:11]: I would just change the readme.md and document that its now possible to install element-js without a .npmrc file ..

**Review by eddyloewen — APPROVED** [review]

**Pull request merged**